pkg/docs/generator.go

```diff
@@ -22,7 +22,12 @@ const DefaultDestination = "./docs/content/cli/"
 func (o *DocsOptions) Validate() error {
 	if o.Destination == "" {
 		o.Destination = DefaultDestination
-	}
+    }
+
+    _, err := os.Stat(o.Destination)
+    if err != nil {
+        return errors.New(fmt.Sprintf("error generating the markdown documentation; desitination '%s' doesn't exist", o.Destination))
```

Comment:
For the error message when we are validating a flag, we try to include the flag name so that the user can tell which flag they need to fix. In this case, changing the error message to "--destination %s doesn't exist" would work.

Also you can simplify this to `errors.Errorf("error message", o.Destination)` The errors package we use, `github.com/pkg/errors` has a nice function to combine those two methods into one.